snap/snapcraft.yaml

```diff
@@ -40,6 +45,15 @@ apps:
       DAEMON_CONFIG_HOME: *daemon-config # temporary
       XTABLES_LIBDIR: $SNAP/usr/lib/$SNAPCRAFT_ARCH_TRIPLET/xtables
     daemon: simple
+    plugs:
+      - kvm
+      - libvirt
+      - multipass-support
+      - network-bind
+      - network-control
+      - network-manager
```

Comment:
I _think_ we should be able to do without this.

Reply:
I only removed the `network-manager` plug here, which is what I think you are referring to.

Reply:
Yup.